src/frontends/tensorflow/src/input_model.cpp

```diff
@@ -336,6 +337,23 @@ std::vector<std::shared_ptr<OpPlace>> InputModel::InputModelTFImpl::topologicall
         ops_to_do.push(output_operation_place);
     }
 
+    // walk through all NextIteration nodes and put their producers into ops_to_do
+    // this is needed to avoid missed nodes in the body graph of TF1 While operation
+    for (const auto& op_place : m_op_places) {
+        auto op_decoder = op_place->get_decoder();
+        if (op_decoder->get_op_type() == "NextIteration") {
+            std::string producer_name;
+            std::string producer_output_port_name;
+            size_t producer_output_port_idx;
+            op_decoder->get_input_node(0, producer_name, producer_output_port_name, producer_output_port_idx);
+            FRONT_END_GENERAL_CHECK(m_op_places_map.count(producer_name),
```

Comment:
You use map.count + map.at (not only here), why don't you use find? It allows to do item lookup only once. Correct me if I miss something in this logic.

Reply:
Yes, you are correct that it will be slightly faster to use iterators but I do not like to work with STL iterators. The code looks worse create iterator, dereference it and use subscript operator.